components/dashboard/src/repositories/detail/ConfigurationDetailVariables.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { ConfigurationVariableList } from "./variables/ConfigurationVariableList";
+import type { Configuration } from "@gitpod/public-api/lib/gitpod/v1/configuration_pb";
+
+type Props = {
+    configuration: Configuration;
+};
+export const ConfigurationDetailVariables = ({ configuration }: Props) => {
+    return <ConfigurationVariableList configuration={configuration} />;
```

Comment:
Do you think this extra layer of components is helpful here? I could see it being useful if we anticipate additional sections on this page. Looking at it as it is, it seems unnecessary though.

Reply:
I think it makes sense. The only reason I originally left it is that like this we have the component for each in the root and only takes care of one thing and that being the content of the entire page. As is, this is not very useful though and I will just point the path to the `ConfigurationVariableList` instead of proxying it through here.